cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -190,8 +190,15 @@
     {# Check and include template-level JavaScript. #}
     {% include '/js/routes/' + request.path.split('/')[1] + '/single.js' ignore missing %}
 
+    {# TODO: Remove conditional when this is fully moved to wagtail. #}
     {# Check and include page-level JavaScript. #}
-    {% include '/js/routes/' + request.path[1:] + 'index.js' ignore missing %}
+    {% if page %}
+        {% for js in page.media %}
+            {% include '/js/routes/on-demand/' + js ignore missing %}
+        {% endfor %}
+    {% else %}
+        `{% include '/js/routes/' + request.path[1:] + 'index.js' ignore missing %}`
```

Comment:
@anselmbradford not actually sure you had included it in the previous PR https://github.com/cfpb/cfgov-refresh/blob/page-json/cfgov/jinja2/v1/_layouts/base.html#L191 figured there was a reason